backends/gstreamer/render-android/lib.rs

```diff
@@ -192,42 +188,37 @@ impl Render for RenderAndroid {
             ));
         }
 
-        let vsinkbin = gst::ElementFactory::make("glsinkbin", Some("servo-media-vsink"))
-            .map_err(|_| PlayerError::Backend("glupload creation failed".to_owned()))?;
-
         let caps = gst::Caps::builder("video/x-raw")
             .features(&[&gst_gl::CAPS_FEATURE_MEMORY_GL_MEMORY])
             .field("format", &gst_video::VideoFormat::Rgba.to_string())
             .field("texture-target", &gst::List::new(&[&"2D", &"external-oes"]))
```

Comment:
When building caps you can get rid of all the `&` here now. None of them are required anymore. Probably also in other places